src/cargo/ops/registry.rs

```diff
@@ -978,6 +978,8 @@ pub struct OwnersOptions {
     pub to_add: Option<Vec<String>>,
     pub to_remove: Option<Vec<String>>,
     pub list: bool,
+    pub subcommand: Option<String>,
```

Comment:
This should probably not be stringly typed.  We should at least use an enum and have the CLI map command names to the enum.

As an alternative, could we instead reuse `to_add` and change the semantics so a `Some(vec![])` infers the package?  That is a breaking change within cargo-the-lib but I don't think it would be for cargo-the-cli which would be fine.

Reply:
Yes, I did. Hope my modification is correct. If there is a problem, I am still willing to continue to modify it.

Reply:
What were your thoughts on this comment of mine?

> As an alternative, could we instead reuse to_add. Personally, that feels like the better route

It seems like the `op` doesn't need to change for this PR

Reply:
No comments were received and no elements were modified at all.